docs/demo/gaiden-css/components/card/centered.html

```diff
@@ -13,21 +13,20 @@
 </head>
 
 <body>
-<article class="card">
-  <header class="card__header content-centered">
-    <h3 class="card__title">Bruce W.</h3>
+
+<article class="card content-centered">
+  <header class="media-box__content media-box__content--card content-centered">
+    <div class="media-box__title">
+      <h3 class="caption-title">
+        Marcos
+        <small>Boa Viagem - Recife</small>
+      </h3>
+    </div>
   </header>
-  <div class="card__content">
-    <p>- Apartamento com pintura nova branca (imóvel com 2 anos). Repintura do forro de gesso e paredes. Forro de gesso da sala com partes sem massa corrida (sanca embutida)</p>
+  <div class="card__content space-element-large">
```

Comment:
@MarcoBrunoBR Great tip! You are right, we don't need the extra div. Updated all demos without it :1st_place_medal: